camel/interpreters/subprocess_interpreter.py

```diff
@@ -141,8 +144,10 @@ def run(
 
         # Print code for security checking
         if self.require_confirm:
-            print(f"The following {code_type} code will run on your computer:")
-            print(Fore.CYAN + code + Fore.RESET)
+            logger.info(
+                f"The following {code_type} code will run on your computer:"
+            )
+            logger.info(Fore.CYAN + code + Fore.RESET)
```

Comment:
I think it doesn't make sense to include Fore.CYAN (or any other formatting from the colorama package) in a logger.info() call